examples/flower-client/client/entrypoint

```diff
@@ -0,0 +1,121 @@
+#!./.flower-example/bin/python
+import os
+import sys
+
+import fire
+
+from fedn.utils.helpers.helpers import get_helper, save_metadata, save_metrics
+
+HELPER_MODULE = "numpyhelper"
+helper = get_helper(HELPER_MODULE)
+
+NUM_CLASSES = 10
+
+
+def _get_node_id():
+    # Figure out FEDn client number from container name
+    number = os.environ.get("CLIENT_NUMBER", "0")
+    return number
+
+
+def _get_flower_client():
+    """Instanziates flower client."""
+    original_argv = sys.argv
+    # Set sys.argv to mimic command-line input required by flower_client.py
+    sys.argv = ["flower_client.py", "--node-id", _get_node_id()]
```

Comment:
I don't get why this is needed? Can you explain a bit?

Reply:
We don't need the actual Flower-client running, right?

Reply:
Thats true, dont need to run the Flower-client, running it complicates the process since It's gets wrapped up in custom flower types that would need to be handled. 

The reason for this code is that I did not want to change anything in the flower file, and it depends on system args. With our code using Fire it's a clash with sys args. So my approach was to programmatically add system args before importing flower, and then resetting back to our Fire args afterwards.

Reply:
I see, ok. Still don't fully get why it would depend on some systems args if we do not make any systems call?

Reply:
The flower code example expects the code the be run with "--node-id" added as an argument. I can work around it as well but then I would need to alter the flower client file a little

Reply:
So the thing to do seems to be to define the input in a config dict instead.